### PR TITLE
Implement new scoring multi platform section

### PR DESCRIPTION
**Pull request by sigurdm**

### CODE REVIEW

File: lib/src/create_report.dart [hunk 1]
```diff
@@ -288,6 +290,75 @@ Future<ReportSection> _followsTemplate(
...
+  List<_Issue> issues;
+  int grantedPoints;
+  if (File(p.join(packageDir, '.dart_tool', 'package_config.json'))
+      .existsSync()) {
+    final tags = <String>[];
+    final explanations = <Explanation>[];
+    final tagger = Tagger(packageDir);
+    final sdkTags = <String>[];
+    final sdkExplanations = <Explanation>[];
+    tagger.sdkTags(sdkTags, sdkExplanations);
```
[the remark this answers is not in the document]
Comment (sigurdm) [reply lib/src/create_report.dart:303]: It is currently not used. Just we need something to pass to the tagger.

Perhaps we should show these though.

File: test/goldens/end2end/angular_components-0.10.0.json [hunk 2]
```diff
@@ -46898,11 +46900,17 @@
...
+        "maxPoints": 20,
+        "summary": "Dart packages: *20/10/0 points:* Supports 2 / 1 / 0 platforms (of Native & JS)\n\nFlutter packages: *20/10/0 points:* Supports 3 / 2 / <1 platforms (of iOS, Android, Web)\n\n\nPackage is a Dart package\nPackage not compatible with runtime native-aot\n\nBecause of the import of dart:html via the import chain package:angular_components/angular_components.dart->package:angular_components/theme/module.dart->package:angular/angular.dart->package:angular/src/platform/browser/tools/tools.dart->package:angular/src/platform/browser/tools/common_tools.dart->package:angular/src/core/linker/component_factory.dart->package:angular/src/core/linker/view_ref.dart->dart:html\nPackage not compatible with runtime native-jit\n\nBecause of the import of dart:html via the import chain package:angular_components/angular_components.dart->package:angular_components/theme/module.dart->package:angular/angular.dart->package:angular/src/platform/browser/tools/tools.dart->package:angular/src/platform/browser/tools/common_tools.dart->package:angular/src/core/linker/component_factory.dart->package:angular/src/core/linker/view_ref.dart->dart:html"
```
[the remark this answers is not in the document]
Comment (sigurdm) [reply test/goldens/end2end/angular_components-0.10.0.json:46907]: Hmm, I'm a bit on the fence here. I think there is value in knowing exactly what file imports what file (and it is the actual imports, not just the dependencies that makes the difference with regards to the tags).

File: lib/src/create_report.dart [hunk 3]
```diff
@@ -288,6 +290,75 @@ Future<ReportSection> _followsTemplate(
...
+    grantedPoints: grantedPoints,
+    summary: _makeSummary(
+      '''
+Dart packages: *20/10/0 points:* Supports 2 / 1 / 0 platforms (of Native & JS)
+
+Flutter packages: *20/10/0 points:* Supports 3 / 2 / <1 platforms (of iOS, Android, Web)
+''',
+      issues,
+      basePath: null,
+      maxIssues:
```
[the remark this answers is not in the document]
Comment (sigurdm) [reply lib/src/create_report.dart:356]: That's actually what I did first, but it seemed arbitrary...
I don't mind too much either way.
Done

File: test/goldens/end2end/angular_components-0.10.0.json (hunk 2 above)
[the remark this answers is not in the document]
Comment (isoos) [reply test/goldens/end2end/angular_components-0.10.0.json:46907]: Oh, I meant keeping the full path, just using the backtick for code formatting and some extra space.

[the remark this answers is not in the document]
Comment (sigurdm) [reply test/goldens/end2end/angular_components-0.10.0.json:46907]: Ah - I can do that!

**Pull request merged**